Tests/StorageTests/ZoomLevelStoreTests.swift

```diff
@@ -0,0 +1,94 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+@testable import Storage
+
+class ZoomLevelStoreTests: XCTestCase {
+    let zoomLevelStore = ZoomLevelStore.shared
+
+    let testHost1 = "www.example1.com"
+    let testHost2 = "www.example2.com"
+    let testHost3 = "www.example3.com"
+    let testZoomLevel: CGFloat = 2.0
+
+    override func tearDown() {
+        super.tearDown()
+        cleanUp()
+    }
+
+    private func cleanUp() {
+        let url = URL(fileURLWithPath: "domain-zoom-levels",
+                      relativeTo: FileManager.documentsDirectoryURL).appendingPathExtension("json")
+        if FileManager.default.fileExists(atPath: url.path) {
+            do {
+                try FileManager.default.removeItem(at: url)
+            } catch {
+                assertionFailure("Error removing file: \(error)")
+            }
+        }
+    }
+
+    func testSaveAndFindZoomLevel() {
+        let domainZoomLevel = DomainZoomLevel(host: testHost1, zoomLevel: testZoomLevel)
+        ZoomLevelStore.shared.save(domainZoomLevel)
+
+        let retrievedZoomLevel = ZoomLevelStore.shared.findZoomLevel(forDomain: testHost1)
```

Comment:
Since you have the variable in the test case we should use it instead of the singleton directly?
```suggestion
        zoomLevelStore.save(domainZoomLevel)

        let retrievedZoomLevel = zoomLevelStore.findZoomLevel(forDomain: testHost1)
```